members/emilKoh.md

```diff
@@ -0,0 +1,16 @@
+# Emil Koh
+
+## Education
+
+* Currently: pursuing Bachelor's in Computer Engineering
+* Previous: Completed course on Computer Fundamentals from NTU
+			GCE 'A' Levels
+
+## Work Experience
+
+* [1 year: 2016 May - Current] Robotics Instructor, at Nullspace Pte Ltd
+* ...
```

Comment:
Can remove this

Reply:
Are we all supposed to include the ## ?

Reply:
The ## is markdown code, that's why the file is a .md